api/filter/accesslog.proto

```diff
@@ -9,24 +9,53 @@ import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
-message AccessLogEntry {
-  // The HTTP request method (RFC 7231/2616)
-  RequestMethod request_method = 1;
+// Envoy access logs describe incoming interaction with Envoy over a fixed
+// period of time, and typically cover a single request/response exchange.
```

Comment:
Should this be "a single request/response exchange or stream (HTTP/gRPC) or connection (TCP)."?

Reply:
Yeah the connection proxy case is sort of the exception here. I can expand the comment to address that.